content/Software Support/Upload/If-your-board-is-not-detected-by-Arduino-IDE.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "If your board is not detected by Arduino IDE"
+id: 4412955149586
+---
+
+Learn what to do if your Arduino board is missing from the board selector or from the _Tools > Port_ menu in Arduino IDE.
+
+> [!NOTE]
+> If you're using Arduino Cloud Editor, see [If your board is not detected by Arduino Cloud Editor](https://support.arduino.cc/hc/en-us/articles/360018131160-If-your-board-is-not-detected-by-Arduino-Cloud-Editor).
+
+## Troubleshooting
+
+### 1. Identify the problem
+
+* **Incorrect Board Identification**: If Arduino IDE incorrectly identifies your board with a generic name or as a different board type, see [If Arduino IDE detects a different board than the one you've connected](https://support.arduino.cc/hc/en-us/articles/12070802257436-If-Arduino-IDE-detects-a-different-board-than-the-one-you-ve-connected) instead.
+
+* **Unknown Devices**: If the board selector in Arduino IDE displays “Unknown” devices, refer to [If there are “Unknown” devices in the board selector and without a board name in the Tools > Port menu](add link) instead.
+
+* **Missing Board**: If your Arduino board is missing from the board selector or from the Tools > Port menu in Arduino IDE, proceed to the instructions below.
```

Comment:
Can we turn this into something like "Check if your board appears as "Unknown" or a board with a different name" and move it down the list? I think this can come after the bootloader reset. This way we keep the beginning of the article as accessible as possible.